Doc/using/venv-create.inc

```diff
@@ -17,10 +19,9 @@ re-used.
 .. versionchanged:: 3.5
    The use of ``venv`` is now recommended for creating virtual environments.
 
-.. deprecated:: 3.6
-   ``pyvenv`` was the recommended tool for creating virtual environments for
-   Python 3.3 and 3.4, and is
-   :ref:`deprecated in Python 3.6 <whatsnew36-venv>`.
+.. deprecated-removed:: 3.6 3.8
+   :program:`pyvenv` was the recommended tool for creating virtual environments
+   for Python 3.3 and 3.4.
```

Comment:
```suggestion
   for Python 3.3 and 3.4, and replaced in 3.5 by executing ``venv`` directly.
```

Perhaps moreso important than giving the history of `pyvenv` is mentioning its replacement (and implicitly its deprecation/removal reason), to avoid requiring the reader to scan around for external contextual information to determine this.